packages/web/components/cards/bond-card.tsx

```diff
@@ -89,14 +90,21 @@ export const BondCard: FunctionComponent<
             </span>
             <div className="flex grow flex-col text-osmoverse-100">
               <h4 className="text-osmoverse-100">
-                {userLockedShareValue.toString()}
+                {<PrivateText text={userLockedShareValue.toString()} />}
               </h4>
               <span className="subtitle1 text-osmoverse-300">
-                {t("pool.sharesAmount", {
-                  shares: formatPretty(userShares.hideDenom(true).trim(true), {
-                    maxDecimals: 4,
-                  }),
-                })}
+                {
```

Comment:
This doesn't need to be in a `{}`. You only need those when you need to evaluate JS, but since `<PrivateText />` is a JSX component, you can remove the `{}`

Reply:
This file was reverted to only keep hiding on the assets page